packages/telegram-cloud-storage-stamper/README.md

```diff
@@ -0,0 +1,195 @@
+# @turnkey/telegram-cloud-storage-stamper
+
+[![npm](https://img.shields.io/npm/v/@turnkey/telegram-cloud-storage-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/telegram-cloud-storage-stamper)
+
+This package contains functions to store a Turnkey API public/private key within [Telegram Cloud Storage](https://core.telegram.org/bots/webapps#cloudstorage). This package also handles stamping a Turnkey request with that API key. It is meant to be used with [`@turnkey/sdk-browser`](https://www.npmjs.com/package/@turnkey/sdk-browser).
```

Comment:
If we're going to mention `@turnkey/sdk-browser` specifically here, it would be optimal to have some integration guide type docs to flesh that out. 

Technically, if this implements the general stamper interface, it could be used with the lowest level `@turnkey/http` as well — we could mention that but generally recommend usage with `@turnkey/sdk-browser`. This also might make sense to... add to `@turnkey/sdk-react` as well?

In any case, a docs update to https://docs.turnkey.com/api-overview/stamps#stampers at the very least seems relevant

Reply:
On the flip side, I can see the argument that we shouldn't because telegram is a pretty separate path and thus shouldn't be included with the other more vanilla browser use cases

Reply:
I specifically removed a reference to @turnkey/http here after our convo during standup yesterday 😅 to push users toward our higher level libraries. I do agree that extra doc will be helpful here, and I'm planning to add some :)